src/Symfony/Component/Translation/DataCollectorTranslator.php

```diff
@@ -88,19 +88,30 @@ public function getCatalogue($locale = null)
     }
 
     /**
-     * Gets the fallback locales.
-     *
-     * @return array $locales The fallback locales
+     * {@inheritdoc}
      */
     public function getFallbackLocales()
     {
-        if ($this->translator instanceof Translator || method_exists($this->translator, 'getFallbackLocales')) {
+        if ($this->translator instanceof TranslatorFallbackInterface) {
             return $this->translator->getFallbackLocales();
         }
 
         return array();
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \BadMethodCallException if the internal translator does not support fallback locales
+     */
+    public function setFallbackLocales(array $locales)
```

Comment:
I think we should drop this method too.

Reply:
I removed it. Anyway I believe it should stay because of <a href="https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L888">Framework Bundle</a>